scripts/generate_driver_wrappers.py

```diff
@@ -163,11 +163,11 @@ def main() -> int:
     """
     Main with command line arguments.
     """
-    def_arg_mbedtls_root = build_tree.guess_mbedtls_root()
+    def_arg_repo_root = build_tree.guess_mbedtls_root()
 
     parser = argparse.ArgumentParser()
-    parser.add_argument('--mbedtls-root', default=def_arg_mbedtls_root,
-                        help='root directory of mbedtls source code')
+    parser.add_argument('--repo-root', default=def_arg_repo_root,
```

Comment:
`scripts/generate_driver_wrappers.py` is a semi-public interface: it's mentioned in the changelog and in the [codegen migration guide](https://github.com/Mbed-TLS/mbedtls/blob/mbedtls-3.5.0/docs/proposed/psa-driver-wrappers-codegen-migration-guide.md#whats-changed), but that guide is a “proposed” interface. If it's a public interface, then arguably its command line options are, since they're documented in the `--help` output.

So is it ok to change its command line options? Users could plausibly rely on build scripts that call `scripts/generate_driver_wrappers.py --mbedtls-root`.

Argument against worrying about this: at the moment, the script is so limited that it's hardly ever usable in production.

Reply:
I agree that this is probably ok to change since the script is in a somewhat transient state. What does @ronald-cron-arm think?

Reply:
Same as @mpg I think it is okay to change it.

Reply:
Cool, nothing to do here then.